DOCUMENT.md

```diff
@@ -47,6 +47,14 @@ kubernetesProvider:
       metadata:
         name: unittest
         namespace: default
+skip:
+    reason: "Unreleased feature"
+postRenderer:
+  cmd: "yq"
```

Comment:
Do we need yq installed? Why could we not rely on golang to do the magic?

YQ is actually written in golang https://github.com/mikefarah/yq

Reply:
The [post-renderer](https://helm.sh/docs/topics/advanced/#post-rendering) is an arbitrary executable.  I used `yq` for the tests because it can easily change yaml for us to assert.  

It could just as easily be a shell script, but I don't trust myself that much with bash.

Reply:
Problem is, there is no way to add this to unit-test framework, hence could be too buggy. And not everyone uses docker image. We are in code, there is a way still to do it

- render chart
- do post rendering

Not an issue, if it get's merged I could to a follow-up improvement